SeaPublicWebsite/Views/EnergyEfficiency/HomeAge.cshtml

```diff
@@ -24,56 +25,80 @@
-                                           <summary class="govuk-details__summary">
-                                               <span class="govuk-details__summary-text">
-                                                   Help me answer this question
-                                               </span>
-                                           </summary>
-                                           <div class="govuk-details__text">
-                                               <p class="govuk-body">
-                                                   You may find information about the age of the property in title deeds or other paperwork you received when
-                                                   you bought your home, or home insurance paperwork.
-                                               </p>
-                                               <p class="govuk-body">
-                                                   If your property is a conversion, enter the year the building was originally built, regardless of
-                                                   what the building was used for before it was your home.
-                                               </p>
-                                               <p class="govuk-body">
-                                                   If your property has been extended, enter the year of the oldest part of the building.
-                                               </p>
-                                           </div>
-                                       </details>
-                                    </text>
-                        }
-                        ))
-                </fieldset>
-            </div>
+                Func<object, object> epcHint = null;
+                if (Model.Epc?.ConstructionAgeBand is not null)
+                {
+                    epcHint = 
+                        @<p class="govuk-body">
+                            Your @Model.Epc?.LodgementDate?.Year Energy Performance Certificate suggests your property was built
+                            <strong>
+                                @(Model.Epc?.ConstructionAgeBand switch {
+                                    HomeAge.Pre1900 or HomeAge.From1900To1929 => "before 1900.",
+                                    HomeAge.From1930To1949 or HomeAge.From1950To1966 => "between 1930 and 1966.",
+                                    HomeAge.From1967To1975 or HomeAge.From1976To1982 => "between 1967 and 1982.",
+                                    HomeAge.From1983To1990 or HomeAge.From1991To1995 => "between 1983 and 1995.",
+                                    HomeAge.From1996To2002 or HomeAge.From2003To2006 => "between 1996 and 2011.",
+                                    HomeAge.From2007ToPresent => "on or after 2012.",
+                                    _ => throw new ArgumentOutOfRangeException()})
```

Comment:
Recommendation service uses this inference logic:
`HomeAge.From2003To2006 => "J",
                    HomeAge.From2007ToPresent => "L",
                    _ => wallConstruction switch`
i.e. epcs for buildings newer than 2007 are treated as band L (i.e.  after 2012

Reply:
Hmm, that may be necessary but it sounds suspicious. Can you check that Joanna is happy with this?

Reply:
Separately, could you put this logic into the view model please - we should start getting into the habit of putting minimal logic into the view itself (view models are much more easily testable)